Simplify plugin settings (#2)
flotiq/flotiq-ui-plugin-multilangual: Readme.md

}
```

> Now, let's consider you are adding two languages: `Poish` and `English`, and your default language is `English`. You are choosing the `title` field to translate. After applying translations, the object will be:

Reviewer: ```suggestion
Now, let's consider that you are adding two languages: `Polish` and `English`, and your default language is `English`. After applying translations, the object will be:
```
Typa, and we no longer choose the fields. Also, the `slug` should be in the `__translations` object.
